lumicks/pylake/kymo.py

```diff
@@ -39,11 +39,14 @@ class Kymo(ConfocalImage):
         End point in the relevant info wave.
     json : dict
         Dictionary containing kymograph-specific metadata.
+    position_offset : float
+        Coordinate position offset.
```

Comment:
I think we should be clear about what the offset is relative to. The most recent parent object that was cropped or the actual raw data?

Reply:
It's the absolute offset. I'll add it to the docstring.

Reply:
I'm unsure if the docstring changes showed up in the most recent version.